src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Validation/ValidateNeverAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// <see cref="IShouldValidate"/> implementation that unconditionally indicates a property should be excluded from
+    /// validation. When applied to a property, the validation system excludes that property. When applied to a type,
+    /// the validation system excludes all properties within that type.
```

Comment:
When applied to a type, system behaves much the same as when an `SuppressChildValidationMetadataProvider` exists for the same type. Should probably recommend `[ValidateNever]` for types users own and `SuppressChildValidationMetadataProvider` for types they don't. For example, MVC doesn't use `SuppressChildValidationMetadataProvider` any place `[ValidateNever]` could be used.